doc/admin-guide/files/records.config.en.rst

```diff
@@ -3721,6 +3734,240 @@ HTTP/2 Configuration
    |TS| gracefully closes connections that have stream error rates above this
    setting by sending GOAWAY frames.
 
+HTTP/3 Configuration
+====================
+
+There is no configuration on this release.
+
+QUIC Configuration
+====================
+
+All configurations for QUIC are still experimental and may be changed or
+removed in the future without prior notice.
+
+.. ts:cv:: CONFIG proxy.config.quic.instance_id INT 0
+   :reloadable:
+
+   A static key used for calculating Stateless Reset Token. All instances in a
+   cluster need to share the same value.
+
+.. ts:cv:: CONFIG proxy.config.quic.connection_table.size INT 65521
+
+   A size of hash table that stores connection information.
+
+.. ts:cv:: CONFIG proxy.config.quic.proxy.config.quic.num_alt_connection_ids INT 65521
+   :reloadable:
+
+   A number of alternate Connection IDs that |TS| provides to a peer. It has to
+   be at least 8.
+
+.. ts:cv:: CONFIG proxy.config.quic.stateless_retry_enabled INT 0
```

Comment:
Kind of off-topic. Should we rename this `proxy.config.quic.server.stateless_retry_enabled` ?

Reply:
Maybe. `proxy.config.quic.stateless_retry_enabled_in` can be another option.